models/staging/stg_account_signers.yml

```diff
@@ -4,6 +4,7 @@ models:
   - name: stg_account_signers
     tests:
       - dbt_utils.recency:
+          tags: [recency]
```

Comment:
Adding this tag will trigger tests on staging models as part of [recency DAG](https://a25091203396483c8baae7abb8cd93c6-dot-us-central1.composer.googleusercontent.com/dags/dbt_recency_tests/grid?root=)

However, recency DAG was designed to only trigger tests for final tables: marts

staging tests are triggered as part of model build. Do we want to trigger staging recency checks apart from model builds?